Qfront/src/Pages/Admin/AddCategory.jsx

```diff
@@ -26,17 +29,21 @@ const AddCategory = () => {
     setError("");
     setSuccess(false);
     //Process data from api call
-    createCategory(user._id, token, { name }).then((data) => {
-      if (data?.error) {
-        setError(true);
-      } else {
-        setName("");
-        setError(false);
-        setSuccess(true);
-      }
-    });
+    const access = token;
+    dispatch(createCategory(user._id, access, { name }))
```

Comment:
this thunk is receiving it's parameters wrongly. Please check to make sure all the thunks are receiving parameters as one argument (could be an object), not as multiple arguments